wcfsetup/install/files/lib/system/user/notification/UserNotificationHandler.class.php

```diff
@@ -239,6 +243,10 @@ public function fireEvent($eventName, $objectType, IUserNotificationObject $noti
 				],
 				'recipients' => $recipients
 			];
+
```

Comment:
Please update the indentation, empty lines use the same amount of tabs, see https://github.com/WoltLab/WCF/blob/master/CONTRIBUTING.md#formatting.

This applies to all other changes in your PR.